src/java.desktop/share/classes/javax/swing/plaf/synth/SynthParser.java

```diff
@@ -213,26 +207,21 @@ class SynthParser extends DefaultHandler {
      *
      * @param inputStream XML document containing the styles to read
      * @param factory DefaultSynthStyleFactory that new styles are added to
-     * @param urlResourceBase the URL used to resolve any resources, such as Images
```

Comment:
The description of this method includes information about the URL

Reply:
The whole sentence is now unnecessary and can be deleted.